File: Program's_Contributed_By_Contributors/Golang Programs/leapYear.go
package main

import "fmt"

func leapYear(n int) {
	if n%4 == 0 {
		if n%100 != 0 {
			fmt.Println("It's a Leap Year")
		} else {
			fmt.Println("Not a Leap Year ")
		}
	} else {
		fmt.Println("Not a Leap Year")
	}
}
func main() {
	leapYear(2020)
}

File: leapyear.py
year = int(input("enter a year: "))
if (year % 4)== 0:
   if (year% 100) == 0:
       if (year% 400) == 0:
          print("%d is a leap year" %year)
       else:
          print("%d is not a leap year" %year)
   else:
       print("%d is a leap year" %year)
else:
   print("%d is not a leap year" %year)